source/Tests/Playnite.Tests/Database/GameDatabaseTests.cs

```diff
@@ -37,6 +37,12 @@ public void ClearPlatforms()
             Platforms.Select(a => a.Id).ToList().ForEach(a => Platforms.Remove(a));
             Assert.AreEqual(0, Platforms.Count);
         }
+
+        public void ClearFilterPresets()

```

Comment:
I'm not sure if this is worth having as a separate method since you are calling it only once.

Reply:
Removed in https://github.com/JosefNemec/Playnite/pull/3258/commits/4524a0500a6fb244b0e527707542db7648baf119